src/fides/core/deploy.py

```diff
@@ -130,17 +130,6 @@ def check_virtualenv() -> bool:
     return True
 
 
-def seed_example_data() -> None:
-    run_shell(
-        DOCKER_COMPOSE_COMMAND
-        + """exec fides /bin/bash -c "fides user login && fides push src/fides/data/sample_project/sample_resources/" """
-    )
-    run_shell(
-        DOCKER_COMPOSE_COMMAND
-        + """exec fides /bin/bash -c "python scripts/load_examples.py" """
```

Comment:
Note to self: I'm inclined to delete these scripts, I think they are most likely just going to surprise us in the future as they are tightly coupled to APIs that evolve over time...